src/use-dropdown-menu.ts

```diff
@@ -81,6 +81,27 @@ export default function useDropdownMenu(itemCount: number) {
 		return () => document.removeEventListener('click', handleEveryClick);
 	}, [isOpen]);
 
+	// Handle listening for item clicks and closing the menu
+	useEffect(() => {
+		// This function is designed to close the menu when an item is clicked
+		const handleItemClick = (event: MouseEvent) => {
+			if ((event.target as HTMLAnchorElement)?.href === undefined) {
+				setIsOpen(false);
+			}
+		};
+
+		if (isOpen) {
```

Comment:
This `if()` statement needs it's own comment, as the previous comment at this level does not still apply here.

```suggestion
		// Add/remove listener based on whether the menu is open
		if (isOpen) {
```